www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -52,6 +56,15 @@
         <Required>N</Required>
         <MinimumValue>1</MinimumValue>
       </server_names_hash_max_size>
+      <ban_response type="OptionField">
+        <multiple>N</multiple>
+        <OptionValues>
+          <http_403 value="403">403 Forbidden</http_403>
+          <http_444 value="444">444 Close</http_444>
```

Comment:
Maybe this way it it easier to understand:
```suggestion
          <http_444 value="444">444 Terminate Connection</http_444>
```

Reply:
sorry, mimiced nginx docs